real_estate/models/test_model.py

```diff
@@ -0,0 +1,33 @@
+from odoo import fields,models
+from datetime import date ,timedelta
```

Comment:
```suggestion
from datetime import date ,timedelta

from odoo import fields,models
```
Generally, we make external imports first and then imports from odoo.
https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html